EDAM_dev.owl

```diff
@@ -61383,6 +61384,53 @@
     </owl:Class>
     
 
+    <!-- http://edamontology.org/topic_4051 -->
+
+    <owl:Class rdf:about="http://edamontology.org/topic_4051">
+        <rdfs:subClassOf rdf:resource="http://edamontology.org/topic_3168"/>
+        <citation rdf:resource="https://doi.org/10.1016/j.cell.2011.11.013"/>
+        <created_in>1.26</created_in>
+        <oboInOwl:creation_date>2025-04-02T14:10:00.851802Z</oboInOwl:creation_date>
+        <oboInOwl:hasDefinition>Chromatin immunoprecipitation-based exeperiment similar to ChIP-seq, with an additional experimental step using an exonuclease to increase the resolution of protein binding site identification.</oboInOwl:hasDefinition>
+        <oboInOwl:inSubset rdf:resource="http://edamontology.org/bio"/>
+        <oboInOwl:inSubset rdf:resource="http://edamontology.org/topics"/>
+        <oboInOwl:savedBy>Claire Rioualen</oboInOwl:savedBy>
+        <rdfs:label>ChIP-exo</rdfs:label>
+        <rdfs:seeAlso rdf:resource="https://en.wikipedia.org/wiki/ChIP-exo"/>
+    </owl:Class>
+
+
+    <!-- http://edamontology.org/topic_4052 -->
+
+    <owl:Class rdf:about="http://edamontology.org/topic_4052">
+        <rdfs:subClassOf rdf:resource="http://edamontology.org/topic_3940"/>
+        <citation rdf:resource="https://doi.org/10.1126/science.1181369"/>
```

Comment:
```suggestion
```

Reply:
same here, relevant "citation" for the technique.